lib/classes/caption/field.php

```diff
@@ -51,11 +55,15 @@ public function __construct(Application $app, databox_field $databox_field, \rec
         foreach ($rs as $row) {
             $this->values[$row['id']] = new caption_Field_Value($this->app, $databox_field, $record, $row['id']);
 
-            // Inconsistent, should not happen
-            if (! $databox_field->is_multi()) {
+            /**
```

Comment:
Should stay as simple comment. PhpDoc style are serialized in opcode, which is useless in this case